### DISCUSSION TITLE
How long does it take vercel to verify a transferred domain?

### DISCUSSION THREAD

**Discussion by Justdealng** [vercel/community#3193] [Help]
### Summary

I transferred my domain to my vercel project and I have added the Domain Verification files to my DNS but still haven't been verified now

### Steps to Reproduce

Added a new Domain to my frontend project
And up till now it hasn't been verified 
Even after adding the TXT file to my Domain Nameservers Records

**Comment by smaeda-ks** [comment]
What's the domain name? I can take a look

**Reply by Justdealng** [reply]
justdeal.ng

The vercel nameservers has been added to the domain

**Reply by smaeda-ks** [reply]
So what we need here is to complete the TXT record verification as the UI instructs. This is because the domain is currently held by another Vercel account. Therefore, you first need to revert the name server changes (i.e., switch back to the original name servers provided by your domain registrar), and then add the verification TXT record via the current domain registrar. This way, we can verify the domain ownership and move the domain to your account. You can then change the name servers to Vercel ones to mange DNS in your Vercel dashboard
